interview-questions.md

```diff
@@ -8,44 +8,55 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What does calling super do in a Ruby class?
 
-Your answer:
+Your answer: Calling a superclass passes information to a subclass that will access data from the superclass. A class inherits from another class.
 
-Researched answer:
+Researched answer: Calling super is used to call the superclass's implementation of a method. When super is invoked within a subclass method, it searches for the same method in the superclass and calls it.
 
 2. What is a gem?
 
-Your answer:
+Your answer: a gem is packaged library that provides a unique type of functionality and can be easily combined with projects. It allows developers to enhance their applications and simplify tasks.
+
+Researched answer: a gem is a packaged library that provides specific functionality and can be easily incorporated into Ruby projects. Gems are essentially collections of code, documentation, and metadata that are distributed and shared within the Ruby community.
 
-Researched answer:
 
 3. What is a relational database? Are there other kinds of databases?
 
-Your answer:
+Your answer: a relational database organizes data into tables with predefined relationships between them. Data is stored in structured tables made up of rows and columns. Other types of databases are Object-Oriented database.
 
-Researched answer:
+Researched answer: A relational database is a type of database that stores and provides access to data points that are related to one another. There are four kinds of this database: document databases, key-value stores, column-oriented databases, and graph databases.
```

Comment:
There are also many other kinds of databases other than relational and object-oriented, be sure to know those!